<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CInputDialogCustomer</class>
 <widget class="QDialog" name="CInputDialogCustomer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>536</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="windowIcon">
   <iconset resource="lava2res.qrc">
    <normaloff>:/images/res/icon.ico</normaloff>:/images/res/icon.ico</iconset>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>500</y>
     <width>981</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
   <property name="centerButtons">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>141</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>Kundennummer:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditCustomerNumber">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>871</width>
     <height>22</height>
    </rect>
   </property>
   <property name="maxLength">
    <number>200</number>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>141</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Name:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditName">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>80</y>
     <width>981</width>
     <height>22</height>
    </rect>
   </property>
   <property name="maxLength">
    <number>200</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>110</y>
     <width>131</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>Vorname:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditFirstName">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>130</y>
     <width>981</width>
     <height>22</height>
    </rect>
   </property>
   <property name="maxLength">
    <number>200</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>160</y>
     <width>131</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>Stra√üe:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditStreet">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>180</y>
     <width>981</width>
     <height>22</height>
    </rect>
   </property>
   <property name="maxLength">
    <number>200</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>210</y>
     <width>121</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>Postleitzahl:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditPostcode">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>230</y>
     <width>221</width>
     <height>22</height>
    </rect>
   </property>
   <property name="maxLength">
    <number>200</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>210</y>
     <width>62</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>Ort:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditCity">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>230</y>
     <width>751</width>
     <height>22</height>
    </rect>
   </property>
   <property name="maxLength">
    <number>200</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>410</y>
     <width>141</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>Kommentar:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditComment">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>430</y>
     <width>981</width>
     <height>21</height>
    </rect>
   </property>
   <property name="maxLength">
    <number>200</number>
   </property>
  </widget>
  <widget class="QLabel" name="labelError">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>470</y>
     <width>981</width>
     <height>20</height>
    </rect>
   </property>
   <property name="palette">
    <palette>
     <active>
      <colorrole role="WindowText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>170</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Text">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>170</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
     </active>
     <inactive>
      <colorrole role="WindowText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>170</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Text">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>170</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
     </inactive>
     <disabled>
      <colorrole role="WindowText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>127</red>
         <green>125</green>
         <blue>123</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Text">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>127</red>
         <green>125</green>
         <blue>123</blue>
        </color>
       </brush>
      </colorrole>
     </disabled>
    </palette>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonNomination">
   <property name="geometry">
    <rect>
     <x>900</x>
     <y>30</y>
     <width>88</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Vorschlag</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditCallnumber">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>280</y>
     <width>981</width>
     <height>22</height>
    </rect>
   </property>
   <property name="maxLength">
    <number>200</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>260</y>
     <width>131</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>Telefonnummer:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditFaxnumber">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>330</y>
     <width>981</width>
     <height>22</height>
    </rect>
   </property>
   <property name="maxLength">
    <number>200</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>310</y>
     <width>131</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>Faxnummer:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditEmail">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>380</y>
     <width>981</width>
     <height>22</height>
    </rect>
   </property>
   <property name="maxLength">
    <number>200</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>360</y>
     <width>131</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>Email:</string>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>lineEditCustomerNumber</tabstop>
  <tabstop>pushButtonNomination</tabstop>
  <tabstop>lineEditName</tabstop>
  <tabstop>lineEditFirstName</tabstop>
  <tabstop>lineEditStreet</tabstop>
  <tabstop>lineEditPostcode</tabstop>
  <tabstop>lineEditCity</tabstop>
  <tabstop>lineEditCallnumber</tabstop>
  <tabstop>lineEditFaxnumber</tabstop>
  <tabstop>lineEditEmail</tabstop>
  <tabstop>lineEditComment</tabstop>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources>
  <include location="lava2res.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>CInputDialogCustomer</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>CInputDialogCustomer</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
